test/coverlet.core.tests/Coverage/InstrumenterHelper.Assertions.cs

```diff
@@ -79,11 +79,46 @@ public static Document Document(this CoverageResult coverageResult, string docNa
             throw new XunitException($"Document not found '{docName}'");
         }
 
+        public static Document Method(this Document document, string methodName)
```

Comment:
Why you need this?I mean the assertion on lines seem enough

Reply:
The problem is that `ExpectedTotalNumberOfBranches()` counts _all_ branches in the whole file. Without the `Method()` filter, I'd have to write `.ExpectedTotalNumberOfBranches(1)` in all three test cases, even when the method I'm testing doesn't have any branches which looks kind of weird. I.e. I'd be asserting things that have nothing to do with my test case.

Reply:
Ok usually I test all three method in same test because are related, btw could be useful for future more complex tests assertions.